### ISSUE TITLE
Add remaining activation messages

### ISSUE THREAD

**Issue by GwnDaan** [ad3154/ISO11783-CAN-Stack#81]
I saw that only the first 4 activation messages were implemented right now (ButtonActivation, SoftKeyActivation, PointingEvent and SelectInputObject). Since there are 11 in total at this time, that means that there are still 7 missing. For the sake of convenience, do we want to implement these as well? I don't mind spending some time on this myself to get this done.

**Comment by GwnDaan** [comment]
I'm talking about the callbacks here

**Comment by ad3154** [comment]
You are correct that support for some of them are missing, and we definitely need to add the others. If you're interested in working on them, that would be very helpful! Things have been busy on my end recently so I've spent a bit less time on this as the holidays grow near.

**Comment by ad3154** [comment]
The numeric value and string ones are probably the most important if I had to guess

**Comment by GwnDaan** [comment]
No worries I'll work on this
> The numeric value and string ones are probably the most important if I had to guess
Yup these are exactly the ones I needed on my end
